tasks/swarm_leave.rb

```diff
@@ -6,8 +6,8 @@
 require 'puppet'
 
 def swarm_leave(force)
-  cmd_string = 'docker swarm leave '
-  cmd_string += ' -f' if force == 'true'
+  cmd_string = ['docker', 'swarm', 'leave']
+  cmd_string += [' -f'] if force == 'true'
```

Comment:
We should try to avoid using += here.

I believe [Array#concat](https://ruby-doc.org/core-2.1.1/Array.html#method-i-concat) would be preferable in this instance.

Also lets rename the variable to cmd rather than cmd_string.

Reply:
See above.

Reply:
In this case you have changed from concatenating a string to an array so we should generally lean towards the more idiomatic approach.